packages/components/src/templates/video-with-description-and-copyright.stories.ts

```diff
@@ -0,0 +1,55 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Video with Description and Copyright',
+  parameters: {
+    chromatic: { disableSnapshot: true }
+  }
+};
+
+export const VideoElementWithDescription = {
+  render: () => html`
+    <sd-video class="sd-media">
+      <video controls="" id="video-example" class="w-[854px] aspect-video">
```

Comment:
Please check if it's possible to remove the id. As far as I see there no need to have it here.

Reply:
Updated